src/app/pages/shared.components/demand/views/completed.demand.view.html

```diff
@@ -151,7 +151,7 @@ <h4 class="modal-title">Demand Details</h4>
                         <div class="row">
                             <div class="form-group col-lg-6 col-md-6 col-sm-4">
                                 <label for="DemandDesc">Demand Description:</label>
-                                <span class="form-control" for="DemandTypeId">{{demand.DemandDesc}}</span>
+                                <span class="form-control" for="DemandTypeId" style="height:100px;">{{demand.DemandDesc}}</span>
```

Comment:
Please create a separate class to apply CSS rules, Please don't use inline style